CHANGELOG.md

```diff
@@ -23,6 +23,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   * `--verbose`: logs + commands/docker output
 - Spinner is started only for a terminal
 
+### Added
+
+- `cartridge replicasets` command to manage replicasets on local running:
```

Comment:
I guess you don't really need `:` at the end of a string

Reply:
Fixed